stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -9,10 +9,12 @@ import Base: length, size, axes, IndexStyle, getindex, setindex!, parent, vec, c
 # but also factorizations, rotations, and other linear algebra objects, including
 # user-defined such objects. so do not restrict the wrapped type.
 """
-    Adjoint(A)
+    Adjoint
```

Comment:
Removed `(A)` to tone down the constructor aspect and to emphasize the type aspect.